src/In.ProjectEKA.HipService/wwwroot/openapi.yaml

```diff
@@ -344,6 +344,9 @@ paths:
     post:
       tags:
         - consents
+      summary: Notify HIP on consent update
+      description: >
+        Handles the notification of the consent based on its status
```

Comment:
Can you explain bit more from API perspective like?
If Status is GRANTED => Consent Artefact and Signature will be available
otherwise => both fields will be empty.